packages/stationary/stationary.0.0.1/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "Izaak Meckler <ihmeckler@gmail.com>"
+authors: "Izaak Meckler <ihmeckler@gmail.com>"
+homepage: "https://github.com/imeckler/stationary"
+bug-reports: "https://github.com/imeckler/stationary/issues"
+license: "MIT"
+dev-repo: "https://github.com/imeckler/stationary.git"
+build: [
+  ["oasis" "setup"]
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "stationary"]
+depends: [
+  "ocamlfind" {build}
+  "oasis" {build}
+  "core" {build}
+  "async" {build}
```

Comment:
Are `core` and `async` really `build` dependencies?  If `stationary` is a library that depends on them then you should drop `{build}` (but keep it for `ocamlfind` and `oasis`).